Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN addgroup -S composer \
     && adduser -S composer -G composer \
     && chown -R composer /opt/apps/laravel-in-kubernetes \
     && apk add --virtual build-dependencies --no-cache ${PHPIZE_DEPS} openssl ca-certificates libxml2-dev oniguruma-dev postgresql-dev \
+    && apk add --no-cache libpq libjpeg-turbo-dev libpng-dev libwebp-dev freetype-dev \
```

Comment:
libpq is already installed below
some library are only required for compiling PHP extension and they no longer needed,
check if this case is true for gd
libpq is required even after the extension is installed in system that's why it is added at the end of script

Reply:
if libjpeg-turbo-dev libpng-dev libwebp-dev freetype-dev are only required to compile then move these depedency at build dependecies